RELEASE/scripts/autoscend/auto_choice_adv.ash

```diff
@@ -374,6 +374,17 @@ boolean auto_run_choice(int choice, string page)
 				run_choice(5);
 			}
 			break;
+		case 1062: // Lots of Options (The Overgrown Lot)
+			if (get_property("questM04Galaktic") != "unstarted" && get_property("questM04Galaktic") != "finished" ) { // update once property is spaded

```

Comment:
could use `if (available_choice_options() contains 1)`
as the choice does not show up if galactic is upstarted or finished

Reply:
Will do. There's bugs with that quest where it changes the properties at the wrong times that I need to report to mafia so this is perfect :+1: